offers/utils.js

```diff
@@ -39,4 +39,16 @@ export const deleteOffer = async ({ t }) => {
         .click(Selector('p').withText('DELETE OFFER'), { offsetX: 200 })
         //close swal - shouldn't we ask about being sure?
         .click(Selector('button').withText('Ok'))
-}
\ No newline at end of file
+}
+
+export const applyForOffer = async({ t, firstName, lastName, email}) => {
+    await t
+        .click(Selector('button').withText('APPLY'))
+        .typeText(Selector('label').withText('Name').parent().find('input'), firstName)
+        .typeText(Selector('label').withText('Surname').parent().find('input'), lastName)
+        .typeText(Selector('label').withText('E-mail address').parent().find('input'), email)
```

Comment:
Wydaje mi się, że dałoby się to uprościć. Patrzysz na input i jaki ma name i wtedy robisz
```js
Selector('input[name=email]')
```
czy coś takiego. W tych plikach z logowaniem tak jest robione i nie wiem, czy to nie jest czytelniejsze, niż ten parent() i find()

Reply:
zasadniczo tak jest, ale z tego co widziałem w kodzie MFA, te inputy akurat nie mają `name`